Spec/RealtimeClient.swift

```diff
@@ -361,6 +361,8 @@ class RealtimeClient: QuickSpec {
                 if let secs = endInterval {
                     expect(secs).to(beLessThanOrEqualTo(UInt(options.suspendedRetryTimeout)))
                 }
+
+                expect(client.connectionStateTtl).to(equal(120 as TimeInterval /*seconds*/))
```

Comment:
Personally I think this test is a little brittle.  If we change our TTL to anything other than 120s, this test will fail.  Why not intercept the CONNECTED protocol message and make sure the `client.connectionStateTtl = connectionStateTtlFromProtocolMessage / 1000)`